src/admin/messages/cn/cmsadmin.php

```diff
@@ -2,219 +2,58 @@
 
```

Comment:
Please don't commit those changes for the cn file, as you have to provide the file inside the `kr` folder and not change the `cn` file.

Reply:
It seems that there was a mistake in using github for the first time I used github. Details Thank you very much. I'll try again as you help me. Thank you.